modules/consul-security-group-rules/main.tf

```diff
@@ -92,3 +101,101 @@ resource "aws_security_group_rule" "allow_dns_udp_inbound" {
   security_group_id = "${var.security_group_id}"
 }
 
+resource "aws_security_group_rule" "allow_server_rpc_inbound_from_security_group_ids" {
+  count                    = "${length(var.allowed_inbound_security_group_ids)}"
```

Comment:
Is `allowed_inbound_security_group_ids` a new input variable for the consul-security-group-rules module? If so, don't you need to add it to its `vars.tf` and pass it in from the `consul-cluster` module?

Reply:
Good catch. I have made these changes.